SPACING.md

```diff
@@ -0,0 +1,335 @@
+# Spacing Style Guide
+When and where to use spaces and blank lines.
+
+### Rational
+(skip if bored)
+
+Humans primarily process and _make sense_ of data when consumed visually. Consistent spacing - and format in general - drastically lowers the cognitive load when reading code and decreases the time needed to understand it. We should consent to a single fixed style of formatting Swift files as part of a journey to better and more readable code. Hence, please enforce these guidelines when reviewing pull requests.
+
+__Note:__ All of the following suggestions are general guidelines. They should be subject to discussions and may be ignored if a convincing reason exists.
+
+---
+
+### Closing Scopes
+Simply __never__ include any whitespace before closing a scope and always __one__ blank line afterwards.
```

Comment:
But indentation is whitespace, too. Maybe write it like this: "When closing a scope, __always__ drop the indentation to the level where the scope originated. Empty scope bodies should be on the same line as their binding (declaration?). Closing a scope is __always__ followed by a newline."

Reply:
This seems unnecessarily strict. For example, I prefer to have two lines before `// MARK:`